### PR TITLE
INT: Temporarily disable "Extract enum variant" intention

### PR DESCRIPTION
**Pull request by mchernyavsky**
bors r+

### CODE REVIEW

File: src/main/kotlin/org/rust/ide/intentions/ExtractEnumVariantIntention.kt [hunk 1]
```diff
@@ -31,15 +29,20 @@ class ExtractEnumVariantIntention : RsElementBaseIntentionAction<ExtractEnumVari
...
-        val variant = element.ancestorStrict<RsEnumVariant>() ?: return null
-        if (variant.isFieldless) {
-            return null
-        }
-        if (variant.variantDiscriminant != null &&
-            ARBITRARY_ENUM_DISCRIMINANT.availability(variant.containingMod) != FeatureAvailability.AVAILABLE) {
-            return null
-        }
-        return Context(variant)
+//        val variant = element.ancestorStrict<RsEnumVariant>() ?: return null
```
Comment (Undin) [inline src/main/kotlin/org/rust/ide/intentions/ExtractEnumVariantIntention.kt]: Maybe it's better just to remove the corresponding line from plugin.xml?

**Comment by mchernyavsky** [comment]
bors retry

**Pull request merged**